### DISCUSSION TITLE
How to extract optimized Airfoil coordinates?

### DISCUSSION THREAD

**Discussion by usmanshahid90** [mdolab/dafoam#116] [General]
I am aiming to plot baseline and optimized airfoil on the same axis to get a visual apple to apple comparison.
The DAFoam FAQ addresses the extraction of optimized geometry, not its coordinates. I have extracted the '.stl '  file but I am unable to extract the coordinates of the optimized geometry.  

Is there any way to get the coordinates of airfoil? 
Kindly guide me. Thanks

**Comment by friedenhe** [comment]
You can use Paraview to open the optimized case, uncheck internal mesh and only show the wing surface, cut an x-y slice for the wing surface, and then choose filter->plot over sorted line. Then you can select the generated filter and choose file save data. The coordinates of the x-y slice will be output to the disk.

**Reply by usmanshahid90** [reply]
Thank you very much sir. It worked really nice
